packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -1517,6 +1517,19 @@ class Positioned extends ParentDataWidget<Stack> {
        bottom = null,
        super(key: key, child: child);
 
+  /// Creates a Positioned object with [left], [top], [right], and [bottom] set
+  /// to 0.0.
+  Positioned.fixedToEdge({
```

Comment:
I like `Position.cover()` minus the fact that it doesn't describe the case when the positioned widget is below non-positioned widgets. How about `framed` or `enframed` or `constrained` or `tight`?

`fromInsets()` is a bit confusing to use without a parameter and `fromInsets(EdgeInsets.zero)` is a little too verbose.